msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -1662,6 +1663,23 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 
 		<!--<Warning Text="_SplitAppExtensionReferencesByExistent: @(_AppExtensionReferenceWithConfigurationExistent)"/>-->
 	</Target>
+	
+	<Target Name="_ExtendAppExtensionReferences" DependsOnTargets="_ResolveAppExtensionReferences">
```

Comment:
The idea here is that after _ResolveAppExtensionReferences we inject the 3rd party extensions into the list being processed later for embedding and code signing.

_ResolvedAppExtensionReferences is an item group of the path, so that's easy.

_AppExtensionCodesignProperties less so. It is generated by reading codesign.items generated by the c# tasks during build, which we don't have. We also don't have a concrete list of types that get serialized, so I dug for every property referenced and set what I believe are sane values.

Reply:
That's fair. I did the PR, and then realized I needed to explain something. 

I figured if it was useful someone would say that :)